scripts/binary/util/upload.js

```diff
@@ -31,41 +31,44 @@ const formHashFromEnvironment = function () {
   throw new Error('Do not know how to form unique build hash on this CI')
 }
 
-const getS3Credentials = function () {
-  const key = path.join('scripts', 'support', 'aws-credentials.json')
-  const config = configFromEnvOrJsonFile(key)
-
-  if (!config) {
-    console.error('⛔️  Cannot find AWS credentials')
-    console.error('Using @cypress/env-or-json-file module')
-    console.error('and filename', key)
-    console.error('which is environment variable', filenameToShellVariable(key))
-    console.error('available environment variable keys')
-    console.error(Object.keys(process.env))
-    throw new Error('AWS config not found')
-  }
+const S3Configuration = {
+  bucket: 'cdn.cypress.io',
+  releaseFolder: 'desktop',
+  binaryZipName: 'cypress.zip',
+  betaUploadTypes: {
+    binary: {
+      uploadFolder: 'binary',
+      uploadFileName: 'cypress.zip',
+    },
+    'npm-package': {
+      uploadFolder: 'npm',
+      uploadFileName: 'cypress.tgz',
+    },
+  },
+}
 
-  la(check.unemptyString(config.bucket), 'missing AWS config bucket')
-  la(check.unemptyString(config.folder), 'missing AWS config folder')
-  la(check.unemptyString(config.key), 'missing AWS key')
-  la(check.unemptyString(config.secret), 'missing AWS secret key')
+const getS3Credentials = async function () {
+  // sso is not required for CirceCI
+  if (process.env.CIRCLECI) {
+    return await fromEnv()()
+  }
 
-  return config
+  return await fromSSO({ profile: process.env.AWS_PROFILE || 'production' })()
```

Comment:
Nit. Technically you don't need to await this.

Reply:
[cypress-io/cypress@`acba7f5` (#20438)](https://github.com/cypress-io/cypress/pull/20438/commits/acba7f5ba8bf7d9a7f026a7e1db069ab73f38982)